enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -629,6 +643,18 @@ func (s *ExecutionServer) execute(req *repb.ExecuteRequest, stream streamLike) e
 		tracing.AddStringAttributeToCurrentSpan(ctx, "execution_result", "merged")
 		tracing.AddStringAttributeToCurrentSpan(ctx, "execution_id", executionID)
 	}
+	// If the action_merger said to hedge this action, run another execution
+	// in the background.
+	if hedge {
+		action_merger.RecordHedgedExecution(ctx, s.rdb, adInstanceDigest)
+		log.CtxInfof(ctx, "Scheduling new hedged execution for %q for invocation %q", downloadString, invocationID)
+		hedgedExecutionID, err := s.Dispatch(ctx, req)
+		if err != nil {
+			log.CtxWarningf(ctx, "Error dispatching execution for %q: %s", downloadString, err)
+			return err
+		}
+		go s.waitExecution(ctx, &repb.WaitExecutionRequest{Name: hedgedExecutionID}, dummyStream{}, waitOpts{isExecuteRequest: true})
```

Comment:
Can you add a log here that records the hedged execution ID?

Reply:
Yup, done